apps/dashboard/services/graphql/config.ts

```diff
@@ -1,4 +1,4 @@
 // export const serverUrl =
 //   process.env.SERVER_URL ?? "https://fe69-93-108-186-220.ngrok-free.app"
 
-export const coreUrl = process.env.CORE_URL ?? "http://localhost:4002/graphql"
+export const coreUrl = process.env.CORE_URL ?? "http://localhost:4002/graphql";
```

Comment:
we should introduce the `env.ts` pattern like we do in `consent` and `core/api` so that all the env variables and default values are been visible from a single file